data/build.gradle

```diff
@@ -43,20 +44,33 @@ dependencies {
       'androidx.appcompat:appcompat:1.0.2',
       'com.google.dagger:dagger:2.24',
       'com.google.protobuf:protobuf-lite:3.0.0',
+      'com.squareup.moshi:moshi-kotlin:1.8.0',
       'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.2',
   )
   testImplementation(
+      'android.arch.core:core-testing:1.1.1',
       'androidx.test.ext:junit:1.1.1',
       'com.google.dagger:dagger:2.24',
       'com.google.truth:truth:0.43',
+      'com.squareup.okhttp3:mockwebserver:4.1.0',
+      'com.squareup.okhttp3:okhttp:4.1.0',
       'junit:junit:4.12',
       'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.2.2',
       'org.mockito:mockito-core:2.19.0',
       'org.robolectric:robolectric:4.3',
   )
+  api(
+      'com.squareup.retrofit2:converter-moshi:2.5.0',
+      'com.squareup.retrofit2:retrofit:2.5.0',
+      'com.squareup.retrofit2:retrofit-mock:2.5.0',
```

Comment:
We shouldn't take a prod dependency on retrofit-mock. Can this be test-only, instead? If not, that's a bit unfortunate, and we should have a separate TODO with a new issue to isolate this to a test dependency to avoid pulling it into production builds.

Reply:
Removed `retrofit-mock` from `api` and also mentioned TODO linked to #59